app/README.MD

```diff
@@ -0,0 +1,26 @@
+# CellScanner 
```

Comment:
readme goed idee; zou hij niet in de root directory moeten? Daar staat een readme maar die is wel heel summier.